apps/entity-service/src/entities/dto/nursery-report.dto.ts

```diff
@@ -0,0 +1,237 @@
+
+  @ApiProperty()
+  status: string;
+
+  @ApiProperty()
+  updateRequestStatus: string;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated project name"
+  })
+  projectName: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated project uuid"
+  })
+  projectUuid: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated organisation name"
+  })
+  organisationName: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated organisation uuid"
+  })
+  organisationUuid: string | null;
+
+  @ApiProperty()
+  updatedAt: Date;
+
+  @ApiProperty({ nullable: true })
+  submittedAt: Date | null;
+
+  @ApiProperty({ nullable: true })
+  taskId: number | null;
+
+  @ApiProperty()
+  dueAt: Date | null;
+
+  @ApiProperty({ nullable: true })
+  title: string | null;
+
+  @ApiProperty({ nullable: true })
+  reportTitle: string | null;
+
+  @ApiProperty()
+  createdAt: Date;
+}
+
+export type AdditionalNurseryReportLightProps = Pick<NurseryReportLightDto, "reportTitle">;
+export type AdditionalNurseryReportFullProps = AdditionalNurseryReportLightProps &
+  AdditionalProps<NurseryReportFullDto, NurseryReportLightDto & Omit<NurseryReport, "nursery">>;
+export type NurseryReportMedia = Pick<NurseryReportFullDto, keyof typeof NurseryReport.MEDIA>;
+
+export class NurseryReportFullDto extends NurseryReportLightDto {
+  constructor(nurseryReport: NurseryReport, props?: AdditionalNurseryReportFullProps) {
+    super();
+    if (nurseryReport != null) {
+      this.populate(NurseryReportFullDto, {
+        ...pickApiProperties(nurseryReport, NurseryReportFullDto),
+        lightResource: false,
+        // these two are untyped and marked optional in the base model.
+        createdAt: nurseryReport.createdAt as Date,
+        updatedAt: nurseryReport.createdAt as Date,
+        ...props
+      });
+    }
+  }
+
+  @ApiProperty({ nullable: true })
+  reportTitle: string | null;
+
+  @ApiProperty({ nullable: true })
+  projectReportTitle: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated nursery name"
+  })
+  nurseryName: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated nursery uuid"
+  })
+  nurseryUuid: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated organisation name"
+  })
+  organisationName: string | null;
+
+  @ApiProperty({
+    nullable: true,
+    description: "The associated organisation uuid"
+  })
+  organisationUuid: string | null;
+
+  @ApiProperty()
+  dueAt: Date | null;
+
+  @ApiProperty()
+  status: string;
+
+  @ApiProperty()
+  updateRequestStatus: string;
+
+  @ApiProperty({ nullable: true })
+  feedback: string | null;
+
+  @ApiProperty({ nullable: true })
+  feedbackFields: string[] | null;
+
+  @ApiProperty()
+  nothingToReport: boolean;
+
+  @ApiProperty()
+  readableCompletionStatus: string;
```

Comment:
This should not be getting calculated in the BE. The FE has everything it needs to map `status` to a readable string.